src/js/background_scripts/message-listener-utils.js

```diff
@@ -222,7 +222,7 @@ https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
                         chrome.windows.update(windowPostUpdate.id, {
                             width: fluid.get(previousProperties, "width") || Math.round(3 * screen.width / 5),
                             height: fluid.get(previousProperties, "height") || Math.round(4 * screen.height / 5),
-                            left: fluid.get(previousProperties, "left") || Math.round(screen.width / 15),
+                            left: fluid.get(previousProperties, "left") || (screen.availLeft + Math.round(screen.width / 15)),
```

Comment:
Seems like you just need add similar logic to the maximise action?

Reply:
Yes, I had replicated the same logic during maximization. But it seems that the background script is returning the equal value of `screen.availLeft` for all monitors (which should be different), and it produced the bug. So, I'm passing the value from content scripts now, and it seems to work.